### DISCUSSION TITLE
Issuer API returns "Sorry, the credential is invalid:Id not whitelisted"

### DISCUSSION THREAD

**Discussion by dirkexe** [Digitaler-Impfnachweis/certification-apis#36] [Q&A] [answered]
Wenn ich den Weg [Authentifizierung mit TI SMCB](https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/SMCB-Authentication.md) gehe und mit dem JWT bei der Issuer API (https://api.ru.impfnachweis.info/api/certify/v2/issue) die Impfbescheinigung anfordere, so wird das mit **"Sorry, the credential is invalid:Id not whitelisted -> Id:191872682"** verweigert. Laut [Spec](https://github.com/Digitaler-Impfnachweis/certification-apis/blob/master/dgc-certify-api.yaml) kann die BSNR der Praxis als ID (`v/0/id`) verwendet werden.

Mit welcher ID kann man in der RU testen?

**Comment by oliver-steinbrecher** [comment]
We're working on a Fix for system with authenticate via SMCB so that there is no whitelisting for those systems.

**Reply by dirkexe** [reply]
Thank you for the quick reply at this late hour. The information was useful to me.

**Reply by dirkexe** [reply]
Is there a timeline by when we can expect this important detail to be fixed? Medical Practitioner offices using this sophisticated SMC-B authentication process, will fail.

**Comment by oliver-steinbrecher** [comment, marked as answer]
We've just deployed a fix for this issue

**Reply by mge-medisoftware** [reply]
The problem remains.  Wich BSNR's are whitelisted in TI RU?
"Sorry, the credential is invalid:Id not whitelisted -> Id:011111100"

**Reply by oliver-steinbrecher** [reply]
We've tested it successfully

**Reply by oliver-steinbrecher** [reply]
please retry

**Reply by dirkexe** [reply]
It is working now. We have been able to successfully test the whole path repeatedly from our software.

**Reply by emelechow** [reply]
@dirkexe Which Id was used finally? We got the same error :-(
